apps/cyberstorm-storybook/public/css/fonts.css

```diff
@@ -0,0 +1,104 @@
+/* sora-100 - latin */
```

Comment:
More code -> more maintaining and hassle. I don't think we need to include support for ancient IE. I also do suspect the overall kB will go down with just the variable font. Even if not using the variable font, I'm thinking just a `.woff2` / `.tff` for each weight would also be enough as they're well supported by modern browsers.

(I mentioned `.ttf` specifically earlier since on a quick glance Google Fonts offered the variable font for Sora as `.ttf`)

My personal opinion is to go for modern approaches wherever possible for us.

Reply:
now using ttf (and woff2)

Reply:
Removed ttf, now only has woff2. Originally left the ttf as it was mentioned in the comment, but seems like woff2 is the way to go.